engines/misc/src/main/java/org/alfresco/transform/misc/transformers/TextToPdfContentTransformer.java

```diff
@@ -130,6 +158,24 @@ public void transform(final String sourceMimetype, final String targetMimetype,
         {
             pageLimit = parseInt(stringPageLimit, PAGE_LIMIT);
         }
+        String pdfFont = transformOptions.get(PDF_FONT);
+        if (pdfFont == null || pdfFont.isBlank())
+        {
+            pdfFont = pdfBoxDefaultFont;
+        }
+        String pdfFontSize = transformOptions.get(PDF_FONT_SIZE);
+        Integer fontSize = null;
+        if (pdfFontSize != null)
+        {
+            try
+            {
+                fontSize = parseInt(pdfFontSize, PDF_FONT_SIZE);
+            }
+            catch (Exception e)
+            {
+                fontSize = DEFAULT_FONT_SIZE;

```

Comment:
Swallowing the exception without logging anything, not even at `DEBUG` level, makes it harder for users and us to troubleshoot issues potentially related to non-parseable `pdfFontSize` values.

Also, if `pdfFontSize` is null we don't initialize the `fontSize` at all. Is this wanted?

Reply:
Added log message in case the font size parsing fails. 

Even if `fontSize` is null, it will be supplied to the `createPDFFromText` method, where the value will be validated and initialized ([here](https://github.com/Alfresco/alfresco-transform-core/blob/1fa2ed033007e7ade468e594e169b735c154a373/engines/misc/src/main/java/org/alfresco/transform/misc/transformers/TextToPdfContentTransformer.java#L297)).